modules/nextflow/src/main/groovy/nextflow/scm/AzureRepositoryProvider.groovy

```diff
@@ -74,8 +78,12 @@ final class AzureRepositoryProvider extends RepositoryProvider {
                 '$format':'json',
                 'path':path
         ] as Map<String,Object>
-        if( revision )
+        if( revision ) {
             queryParams['versionDescriptor.version']=revision
+
+            if( COMMIT_REGEX.matcher(revision).matches() )
```

Comment:
Wonder if it should be supported also partial commit ie. shorter than 40 chars?

Reply:
I tried, it is not